### DISCUSSION TITLE
Lighthouse Test: Page Performance, Remove Unused JavaScript

### DISCUSSION THREAD

**Discussion by bsor-dev** [nuxt/nuxt#9264] [General]
Is this really unused file or the lighthouse misleading? Some people say these files are actually used. I'm using universal (SSR) I can't believe my app performance is only 0 to 15

![image](https://user-images.githubusercontent.com/26837762/117616975-791fac00-b19e-11eb-987e-4836f098c6c6.png)

**Comment by danielroe** [comment]
Let's move discussion to https://github.com/nuxt/nuxt.js/discussions/8849 to keep helpful comments in one place 🙃
